app/assets/javascripts/search_toggle.js

```diff
@@ -0,0 +1,16 @@
+$(document).on('ready page:load', function(e) {
+    $("#google_search").click(function() {
+        // Replace search icon with close
+        if($('#google_search_wrapper').css('display') == "none") {
+            $('#google_search i').removeClass('fa-search').addClass('fa-times');
+            $('#nav').css('z-index', 1000);
+        } else {
+            $('#google_search i').removeClass('fa-times').addClass('fa-search');
+        }
+
+        $('#google_search_wrapper').fadeToggle(200, function() {
+            $("#gsc-i-id1").focus();
+        });
+    });
+    return false;
+});
```

Comment:
Okay, so I would like you to try this:

```
function initGoogleSearch() {
   $("#google_search").click(function() {
         // Replace search icon with close
         if($('#google_search_wrapper').css('display') == "none") {
             $('#google_search i').removeClass('fa-search').addClass('fa-times');
             $('#nav').css('z-index', 1000);
         } else {
             $('#google_search i').removeClass('fa-times').addClass('fa-search');
         }
 
         $('#google_search_wrapper').fadeToggle(200, function() {
             $("#gsc-i-id1").focus();
         });
     });
     return false;
}

$(document).ready(function () {
    initGoogleSearch()
});
document.addEventListener('page:load', function () {
   initGoogleSearch()
});
```

Have not tested it, but... yeah, le's see if it works